docs/modules/ROOT/pages/reference/extensions/core.adoc

```diff
@@ -174,7 +174,7 @@ What to do if it is not possible to extract CSimple expressions from a route def
 
 |icon:lock[title=Fixed at build time] [[quarkus.camel.main.enabled]]`link:#quarkus.camel.main.enabled[quarkus.camel.main.enabled]`
 
-If `true` all `camel-main` features are enabled; otherwise no `camel-main` features are enabled. See described the xref:user-guide/configuration.adoc#_camel_main[Bootstrap] section of Camel Quarkus documentation for more details.
+If `true` all `camel-main` features are enabled; otherwise no `camel-main` features are enabled. See described the xref:user-guide/bootstrap.adoc#_camel_main[Bootstrap] section of Camel Quarkus documentation for more details.
```

Comment:
I thought bootstrap.adoc was removed. Am I mixing commits ?

Reply:
Yes, good catch, this actually comes from the config POJO JavaDoc. I have fixed it.